frontend/src/components/SeedlotRegistrationProgress/InterimStorage/index.tsx

```diff
@@ -0,0 +1,371 @@
+              disableWheel
+              hideSteppers
+              type="text"
+              label="Interim agency location code"
+              helperText="2-digit code that identifies the address of operated office or division"
+              invalid={invalidNumber}
+              invalidText="Please enter a valid value"
+              readOnly={isChecked}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => inputChangeHandlerStorage(e)}
+              onBlur={() => validateInterimAgencyLocationNumber()}
+            />
+          </Column>
+        </Row>
+        <Row className="storage-information-title">
+          <Column lg={8}>
+            <h2>Storage information</h2>
+            <Subtitle text="Enter the interim storage information for this lot" />
+          </Column>
+        </Row>
+        <Row className="storage-date-row">
+          <Column className="start-date-col" sm={4} md={2} lg={4}>
+            <DatePicker
+              datePickerType="single"
+              name="startDate"
+              maxDate={mockStartDataMax}
+              minDate={mockStartDataMin}
+              onChange={(e: any) => inputChangeHandlerDate(e, 'startDate')}
+            >
+              <DatePickerInput
+                id="start-date-input"
+                labelText="Collection start date"
+                helperText="year/month/day"
+                placeholder="yyyy/mm/dd"
+              />
+            </DatePicker>
+          </Column>
+          <Column className="end-date-col" sm={4} md={2} lg={4}>
+            <DatePicker
+              datePickerType="single"
+              name="endDate"
+              maxDate={mockEndDataMax}
+              minDate={mockEndDataMin}
+              onChange={(e: any) => inputChangeHandlerDate(e, 'endDate')}
+            >
+              <DatePickerInput
+                id="end-date-input"
+                labelText="Collection end date"
+                helperText="year/month/day"
+                placeholder="yyyy/mm/dd"
+              />
+            </DatePicker>
+          </Column>
+        </Row>
+        <Row className="storage-location-row">
+          <Column sm={4} md={4} lg={8}>
+            <TextInput
+              id="storage-location-input"
+              name="location"
+              ref={storageLocationInputRef}
+              type="text"
+              labelText="Storage location"
+              placeholder="Enter the location were the cones were stored"
+              helperText="Enter a short name or description of the location where the cones are being temporarily stored"
+              invalid={storageLocationEmpty}
+              invalidText="Please enter a valid value"
+              onBlur={() => validateStorageLocation()}
+              onChange={(e: React.ChangeEvent<HTMLInputElement>) => inputChangeHandlerLocation(e)}
+            />
+          </Column>
+        </Row>
+        <Row className="cl/ass-source-radio">
```

Comment:
typo in className`cl/ass`

Reply:
Also, the class name doesn't relate to row functionality. Maybe something like `storage-tipe-radio`?